packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -54,63 +54,168 @@ Therefore, when dealing with Custom Roles, these three entities always exist:
 
 ### Resource sets
 
-A Resource Set is simply a collection of resources. The following resources are currently supported:
+A Resource Set is simply a collection of resources. There are two types of resource identifiers, resources can either be identified by Okta Resource Names (ORN) or by a REST URL format.
 
-* All Users
-* All Groups
-* A specific Group
-* All Users within a specific Group
-* All Apps
-* All Apps of the same type
-* A specific App
+#### Okta Resource Name (ORN)
 
-### Identifiers
+<ApiLifecycle access="ea" />
 
-#### Resource identifiers
-
-To specify a resource targeted by a Resource Set, you can use the REST URL of the corresponding Okta API:
-
-* [All Users](/docs/reference/api/users/#list-users)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/users
-  ```
-
-* [All Groups](/docs/reference/api/groups/#list-groups)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/groups
-  ```
-
-* [All Apps](/docs/reference/api/apps/#list-applications)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/apps
-  ```
-
-* [A specific Group](/docs/reference/api/groups/#get-group)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}
-  ```
-
-* [All Users within a specific Group](/docs/reference/api/groups/#list-group-members)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}/users
-  ```
-
-* [All Apps of specific type](/docs/reference/api/apps/#list-apps-by-name)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/apps/?filter=name+eq+\"${targetAppType}\"
-  ```
-
-* [A specific App](/docs/reference/api/apps/#get-application)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/apps/${targetAppId}
-  ```
+Okta Resource Names (ORN) is the primary resource identifier. ORNs uniquely identify Okta resources. Currently ORNs are available in early access.
+
+##### ORN Format
+
+ORNs generally follow a format, however each type of resource have their own specific format.
+
+`orn:okta:service:service-specific-id`
```

Comment:
I gave similar feedback to Vasi on this doc: https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2514163154/ORNs+in+public+APIs, that while theoretically this may be the case for the format, 99% of ORNs customer encounter will be of the form: orn:{partition}:{service}:{tenantId}:{objectType}:{objectId}
We should be consistent with that and introduce them as qualified to a tenantId/orgId from the start to make it easier for customers/reader to see the pattern and generalize.